labex/src/components/AdmCreateTripPage.js

```diff
@@ -0,0 +1,133 @@
+import React, { useState } from 'react'
+import {PageStyle, ActualPage, Inputs, Button, BackButton} from  './styled'
+import axios from 'axios'
+
+import NavBar from './NavBar'
+
+    const useForm = initialValues => {
+        const [form, setForm] = useState(initialValues);
+        const onChange = (name, value) => {
+            const newForm = {...form, [name]: value}
+            setForm(newForm)
+        }
+        return {form, onChange}
+    }
+const FormPage = () => {
+
+    const {form, onChange} = useForm({name:"", planet: "", date:"", description:"", durationInDays:""})
+    
+    const handleInputChange = (event) => {
+        const {name, value} = event.target;
+        onChange(name, value)
+    }
+
+    const createTrip = () => {
+
+        const token = window.localStorage.getItem('token')
+        
+        const axiosConfig = {
+            headers : {auth: token}
+        }
+        axios
+            .post("https://us-central1-labenu-apis.cloudfunctions.net/labeX/lais-mello/trips", form , axiosConfig)
+            .then(response => {
+                console.log(response)
+                console.log(form)
+            })
+            .catch(e => {
+                console.log(e)
```

Comment:
Apesar de console.log(response) funcionar na linha 33, o console.log(e) da linha 38 irá imprimir um erro sem informações. Para corrigir esse problema, basta acessar a propriedade response dentro do _**e**_.
```suggestion
                console.log(e.response)
```